cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/ZookeeperScalerTest.java

```diff
@@ -180,20 +176,14 @@ public void testConnectionTimeout(VertxTestContext context)  {
         ZooKeeperAdmin mockZooAdmin = mock(ZooKeeperAdmin.class);
         when(mockZooAdmin.getState()).thenReturn(ZooKeeper.States.NOT_CONNECTED);
 
-        ZooKeeperAdminProvider zooKeeperAdminProvider = new ZooKeeperAdminProvider() {
-            @Override
-            public ZooKeeperAdmin createZookeeperAdmin(String connectString, int sessionTimeout, Watcher watcher, ZKClientConfig conf) throws IOException {
-                return mockZooAdmin;
-            }
-        };
+        ZooKeeperAdminProvider zooKeeperAdminProvider = (connectString, sessionTimeout, watcher, conf) -> mockZooAdmin;
```

Comment:
Wasn't the original code more readable? I have no idea what does this mean.

Reply:
`ZookeeperAdminProvider` is a function-like/SAM interface, so the new version is shorthand of the old version.

Reply:
I don't know ... for me the old thing was much easier to read and understand without thinking about what it actually does. What was wrong with it? Did IntelliJ suggested to safe 4 lines in the file?